<!-- Launches oculus_publisher, oculus_subscriber, camera publisher, dynamic reconfigure gui
     with default sonar parameters (as defined in SimpleFireMessage.cpp), and rqt_gui
-->
<launch>
  <param name="initRange" value="2"/>
  <param name="initGainPercent" value="50"/>
  <param name="initGamma" value="127"/>
  <param name="initPingRate" value="2"/>
  <param name="initMasterMode" value="2"/>
  <node pkg="oculus_sonar_ros" type="oculus_publisher" name="oculus_publisher" output="screen"/>
  <node pkg="oculus_sonar_ros" type="oculus_subscriber" name="oculus_subscriber"/>
  <node pkg="blackmagic_ros" type="blackmagic_publisher" name="blackmagic_publisher" output="screen" required="true"/>
  <node pkg="rqt_gui" type="rqt_gui" name="rqt_gui" args="--perspective-file $(find oculus_sonar_ros)/rqt_config/sonar_and_cameras.perspective"/>
</launch>
